js/ui/lookup.d.ts

```diff
@@ -201,7 +228,7 @@ export interface dxLookupOptions extends dxDropDownListOptions<dxLookup> {
      * @prevFileNamespace DevExpress.ui
      * @public
      */
-    onValueChanged?: ((e: { component?: dxLookup, element?: TElement, model?: any, value?: any, previousValue?: any, event?: TEvent }) => any);
+    onValueChanged?: ValueChangedEvent<dxLookup>;
```

Comment:
Is this a typo? I guess it should be
```suggestion
    onValueChanged?: ((e: ValueChangedEvent<dxLookup>) => any);
```